web/modules/weather_data/src/Service/HourlyForecastTrait.php

```diff
@@ -254,7 +254,7 @@ public function getHourlyForecastFromGrid(
 
             return [
                 "apparentTemperature" =>
-                    abs($apparentTemperature - $temperature) >= 5
+                    abs($apparentTemperature - $temperature) >= 0
```

Comment:
I kept in the ternary if we want to edit this later, but if that's not a great practice happy to adjust it so it's just the variable `$apparentTemperature`